diff -Nur Sunflower-develop/sunflower/gui/main_window.py Sunflower-develop-b/sunflower/gui/main_window.py
--- Sunflower-develop/sunflower/gui/main_window.py	2020-10-03 22:31:01.000000000 +0600
+++ Sunflower-develop-b/sunflower/gui/main_window.py	2020-10-05 17:10:16.222109533 +0600
@@ -916,13 +916,13 @@
 		# try loading from zip file
 		if os.path.isfile(sys.path[0]) and sys.path[0] != '':
 			archive = zipfile.ZipFile(sys.path[0])
-			with archive.open('styles/main.css') as raw_file:
+			with archive.open('/usr/share/sunflower/styles/main.css') as raw_file:
 				provider.load_from_data(raw_file.read())
 			archive.close()
 
 		# load styles from a file
 		else:
-			file_name = os.path.join(common.get_static_assets_directory(), 'styles', 'main.css')
+			file_name = os.path.join('/usr/share/sunflower/styles/main.css')
 			provider.load_from_file(Gio.File.new_for_path(file_name))
 
 		# apply styles
